<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateCal</class>
 <widget class="QWidget" name="DateCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDateEdit" name="dateEdit_start"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>差值</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendar_start"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>日期之间的相隔时间</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>开始时间</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEdit_end"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget_2</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit_end</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendar_start</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit_start</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
